apps/dh/app-dh/src/configuration/load-dh-b2c-environment.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  DhB2CEnvironment,
+  loadDhB2CEnvironment as _loadDhB2CEnvironment,
+} from '@energinet-datahub/dh/shared/environments';
+
+export function loadDhB2CEnvironment(): Promise<DhB2CEnvironment> {
+  return _loadDhB2CEnvironment('dh-b2c-environment.json').catch(() =>
+    _loadDhB2CEnvironment('dh-b2c-environment.local.json')
+  );
+}
```

Comment:
suggestion: add a `fileReplacements` entry in `apps\dh\app-dh\project.json` to replace `load-dh-b2c-environment.ts` with `load-dh-b2c-environment.prod.ts` on production build.

Reply:
Good catch!